esphome/components/emporia_vue/emporia_vue.h

```diff
@@ -42,16 +42,16 @@ struct __attribute__((__packed__)) SensorReading {
 };
 
 class PhaseConfig;
-class CTSensor;
+class CTClampConfig;
 
 class EmporiaVueComponent : public Component, public i2c::I2CDevice {
  public:
   void dump_config() override;
 
   void set_sensor_poll_interval(uint32_t sensor_poll_interval) { this->sensor_poll_interval_ = sensor_poll_interval; }
   uint32_t get_sensor_poll_interval() const { return this->sensor_poll_interval_; }
-  void set_phases(std::vector<PhaseConfig *> phases) { this->phases_ = std::move(phases); }
-  void set_ct_sensors(std::vector<CTSensor *> sensors) { this->ct_sensors_ = std::move(sensors); }
+  void set_phases(std::vector<PhaseConfig *> phases) { this->phases_ = phases; }
```

Comment:
```suggestion
  void set_phases(std::vector<PhaseConfig *> phases) { this->phases_ = std::move(phases); }
```
Can you please reverse this.